### PR TITLE
🌱 Uplift CAPI to v1.8.0-rc.1

### PR DESCRIPTION
**Pull request by Sunnatillo**
Bumps CAPI  to v1.8.0-rc.1

### CODE REVIEW

File: docs/e2e-test.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ in the ephemeral cluster either before pivoting or after re-pivoting.
 
 | tests         | CAPM3 from  | CAPM3 to  | CAPI from  | CAPI to         |
 | --------------| ----------- | --------- | ---------- |---------------- |
-| v1.7=>current | v1.7.0      | main      | v1.7.3     | latest release  |
+| v1.7=>current | v1.7.0      | main      | v1.8.0-rc.1    | latest release  |
```
Comment (tuminoid) [inline docs/e2e-test.md:104]: CAPI from 1.8.0-rc.1? This doesn't sound right.

**Review by kashifest — COMMENTED** [review]
/approve

File: docs/e2e-test.md (hunk 1 above)
Comment (kashifest) [reply docs/e2e-test.md:104]: ah true, lets not touch these things until GA is there
/hold

Comment (Sunnatillo) [reply docs/e2e-test.md]: Nice catch. Fixed now.

File: docs/e2e-test.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ in the ephemeral cluster either before pivoting or after re-pivoting.
 
 | tests         | CAPM3 from  | CAPM3 to  | CAPI from  | CAPI to         |
 | --------------| ----------- | --------- | ---------- |---------------- |
-| v1.7=>current | v1.7.0      | main      | v1.7.3     | latest release  |
+| v1.7=>current | v1.7.0      | main      | v1.7.3    | latest release  |
```
Comment (tuminoid) [inline docs/e2e-test.md:104]:
```suggestion
| v1.7=>current | v1.7.0      | main      | v1.7.3    | latest release   |
```
Put the space back so we avoid changing the line and keep formatting intact.

**Comment by Sunnatillo** [comment]
/override metal3-centos-e2e-integration-test-main
/override metal3-ubuntu-e2e-integration-test-main

The tests passed on previously. I did one space change.

**Review by tuminoid — COMMENTED** [review]
/lgtm

**Pull request merged**